semantic/src/site/globals/site.overrides

```diff
@@ -9,6 +9,7 @@ body {
 p {
   font-size: @fontMedium;
   letter-spacing: 0;
+  padding: 0;
```

Comment:
Hmm I was thinking that it might be easier for us since this is defined everywhere - and each place will have a slightly different padding, I think this is alright for now, and we can check we needed special cases after.

Reply:
Okay, sure.